### PR TITLE
Plateu of Rex / Zachariel Crossroads / Royal Mausoleum

### PR DESCRIPTION
**Pull request by terrapin47**

### CODE REVIEW

File: ITEM.tsv [hunk 1]
```diff
@@ -5687,7 +5687,7 @@ ITEM_20150717_005686	There is a story that the first person who received these s
 ITEM_20150717_005687	This equipment was adopted from Gele Plateau Area. The history of this item is long so it is believed to have been made by the first Paladin Master
 ITEM_20150717_005688	A strong point of mounted equipment by monsters is to borrow the monster's abilities.
 ITEM_20150717_005689	Nepenthes is not an active monster. Then, the feature of this equipment is a reaction of the origin.
-ITEM_20150717_005690	Originally, Schnaimm is the creation contained with a plan and arrangement. Anyway, it would be a good thiing if it helps people.
+ITEM_20150717_005690	Originally, Shnayim was a creation bearing king Zachariel's great plans and arrangements. As for now, it would be of more help this way.
```
Comment (Sunaries) [inline ITEM.tsv]: Great King Zachariel

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -1877,16 +1877,16 @@ QUEST_LV_0100_20150317_001872	$Oh, I didn't know I could count on you.{nl}I will
...
 QUEST_LV_0100_20150317_001881	$Hold on for a minute. I've seen this before from a book.{nl}Wasn't Rexipher a Demon Lord who is part of the army of the Demon King, Vaiga?
-QUEST_LV_0100_20150317_001882	Using the name of a demon, that doen't look like a good intent.
-QUEST_LV_0100_20150317_001883	$Rexipher?{nl} The one with silver hair and wearing a robe?
-QUEST_LV_0100_20150317_001884	$He is a crazy person who can control monsters.{nl}He made me like this and also destroyed the altar which I used for research.{nl}
-QUEST_LV_0100_20150317_001885	$First, go to the Chesed Altar. {nl}They are also aiming for the Gedulah Altar.
+QUEST_LV_0100_20150317_001882	Using the name of a demon, that doesn't look like they have good intentions.
+QUEST_LV_0100_20150317_001883	$Rexipher?{nl} The one with silver hair, wearing a robe?
+QUEST_LV_0100_20150317_001884	$He is a crazy person who can control monsters.{nl}He's the reason I'm wounded like this, and he also destroyed the altar which I used for research.{nl}
+QUEST_LV_0100_20150317_001885	$He's attacking Chesed Altar, go there first. {nl}He is also going for Gedulah Altar.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Do you see him attacking at the Chesed Altar?

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -5299,7 +5299,7 @@ QUEST_LV_0100_20150317_005294	{memo X}
 QUEST_LV_0100_20150317_005295	{memo X}
 QUEST_LV_0100_20150317_005296	{memo X}
 QUEST_LV_0100_20150317_005297	Go to Cyrenia Odell
-QUEST_LV_0100_20150317_005298	The Cetek Altar was destroyed but you managed to release the seal of Sviesa Altar. Talk to Cyrenia Odell.
+QUEST_LV_0100_20150317_005298	The Chesed Altar was destroyed, but you managed to release the seal of Viesha Altar. Talk to Cyrenia Odell.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Hm, the originals says 스에비사 which is Sviesa. 비에샤 is Viesha.

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -5865,7 +5865,7 @@ QUEST_LV_0100_20150317_005860	Found the Treasure Chest but it is empty and you j
 QUEST_LV_0100_20150317_005861	What the guard is searching for
 QUEST_LV_0100_20150317_005862	Told him that the treasure chest was empty. But the guard still seems like he needs more help. Talk to him.
 QUEST_LV_0100_20150317_005863	Find necklace from the Hogmas
-QUEST_LV_0100_20150317_005864	When you use the V key, the body of hogma leader with the necklace will shine. Find the necklace and give it to the guard.
+QUEST_LV_0100_20150317_005864	When you use the V key, the body of the Hogma leader with the necklace will shine. Find the necklace and give it to the guard.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Hogma leader -> Hogma Captain

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -6551,7 +6551,7 @@ QUEST_LV_0100_20150323_006546	Defeat the monsters in Owl Grave
 QUEST_LV_0100_20150323_006547	Defeat the monsters in Lemprasa Pond
 QUEST_LV_0100_20150323_006548	Officer Liudas say he'll give you an accessory if you defeat the monsters around. Defeat the monsters around Lamprasa Pond.
 QUEST_LV_0100_20150323_006549	Defeated a lot of monsters in Lamprasa Pond. Report to Offier Liudas. 
-QUEST_LV_0100_20150323_006550	Historian Collin wants to honor the exploere who died while exploring this area. Burn the oration in their honor in front of the epitaph in Rukas Plateau.
+QUEST_LV_0100_20150323_006550	Historian Collin wants to honor the exploerers who died while exploring this area. Burn the oration in their honor in front of the epitaph at Rukas Plateau.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: exploerers -> explorer

File: QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -6889,7 +6889,7 @@ QUEST_LV_0100_20150511_006884	Varkis' Spirit
 QUEST_LV_0100_20150511_006885	{memo X}That is my adventure journal.. lots of my diaries are hidden at Rukas Plateau besides that..{nl}I want you to retrieve those..
 QUEST_LV_0100_20150511_006886	{memo X}That object is not useful to me anymore now..{nl}If you could find those diaries, I will make you the completed version.
 QUEST_LV_0100_20150511_006887	{memo X}Here it is..{nl}The completed version.. I will leave now..
-QUEST_LV_0100_20150511_006888	The one who fell seeking for knowledge.{nl}Gone back to the Goddess by becoming a part of the canyon.{nl}Saddened mind would penetrate the gold stones, the liquor that is poured would permeate onto the ground for tears..{nl}
+QUEST_LV_0100_20150511_006888	The one who fell while researching..{nl}Gone back to the Goddess by becoming a part of the canyon.{nl}Saddened mind would penetrate the gold stones, the liquor that is poured would permeate onto the ground for tears..{nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: while researching/seeking for knowledge.

File: QUEST_LV_0100.tsv [hunk 7]
```diff
@@ -6939,9 +6939,9 @@ QUEST_LV_0100_20150714_006934	{memo X}My soul is shredded and trapped in the sea
 QUEST_LV_0100_20150714_006935	The Holy Arc that had me inside is broken and the observers obtained those pieces.{nl}Please defeat them and complete the Holy Arc.
 QUEST_LV_0100_20150714_006936	Our guards love to fight.{nl}The Pantos were not this reckless before. {nl}I wonder if we should be like this when the demons are waiting to march in.
 QUEST_LV_0100_20150714_006937	If possible, I want you to defeat the Wild Carnivore as well.{nl}It is corrupted with the demonic energy and I can't stand it anymore.
-QUEST_LV_0100_20150714_006938	Tools for stonemasons? {nl}10,000 Silvers will do.{nl}{nl}
+QUEST_LV_0100_20150714_006938	Tools for stonemasons? {nl}10,000 Silver will do.{nl}{nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Silver -> silver? Since I believe we agreed on this.

File: QUEST_LV_0100.tsv (hunk 2 above)
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: I was going off of the quest text, what does the original say?  The quest text mentions that he's attacking it, so I'd like to change that too if it's not accurate.

File: QUEST_LV_0100.tsv (hunk 3 above)
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: Just what the wiki says

File: QUEST_LV_0100.tsv (hunk 2 above)
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: 'First, go to the Chesed Altar.' is what it really says.

**Comment by terrapin47** [comment]
This is good to merge :+1:

File: QUEST_LV_0100.tsv (hunk 6 above)
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: The term 'researching' is ambiguous. The original says he is exploring for knowledge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by terrapin47** [comment]
Resolved conflicts.

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**